pkg/operator/ceph/object/spec.go

```diff
@@ -407,36 +414,52 @@ func configureReadinessProbe(container *v1.Container, healthCheck cephv1.ObjectH
 }
 
 func (c *clusterConfig) defaultLivenessProbe() *v1.Probe {
```

Comment:
Need to remove the liveness probe from the object docs, example yamls, types.go, etc?